### PR TITLE
CRVV1ETHSTETH

### PR DESCRIPTION
**Pull request by talbaneth**

### CODE REVIEW

File: src/MegaPoker.sol [hunk 1]
```diff
@@ -34,6 +34,7 @@ contract PokingAddresses {
     address constant univ2wbtcdai   = 0x5bB72127a196392cf4aC00Cf57aB278394d24e55;
     address constant matic          = 0x8874964279302e6d4e523Fb1789981C39a1034Ba;
     address constant wsteth         = 0xFe7a2aC0B945f12089aEEB6eCebf4F384D9f043F;
+    address constant crvv1ethsteth  = 0x2b465de3e69a2EC00158F0E3B4614e3582430ab2;
```
Comment (hexonaut) [inline src/MegaPoker.sol:37]: This has been redeployed: `0x0A7DA4e31582a2fB4FD4067943e88f127F70ab39 `

**Review by hexonaut — CHANGES_REQUESTED** [review]
New oracle has `src()`. Should update to that. I think OmegaPoker should grab that automatically now.

File: src/MegaPoker.sol (hunk 1 above)
Comment (talbaneth) [reply src/MegaPoker.sol]: Done

File: src/MegaPoker.sol [hunk 2]
```diff
@@ -61,6 +62,7 @@ contract MegaPoker is PokingAddresses {
         (ok,) = univ2wbtcdai.call(abi.encodeWithSelector(0x18178358));
         (ok,) = matic.call(abi.encodeWithSelector(0x18178358));
         (ok,) = wsteth.call(abi.encodeWithSelector(0x18178358));
+        (ok,) = crvv1ethsteth.call(abi.encodeWithSelector(0x18178358));
```
Comment (brianmcmichael) [inline src/MegaPoker.sol:65]: I've pinged OCU about this. We might be able to get away with poking this once a day like the GUNI tokens.

Comment (brianmcmichael) [reply src/MegaPoker.sol:65]: nvm price is in fiat :sweat_smile:

**Comment by brianmcmichael** [comment]
Changes look good. 

We'll want to deploy and then update the megapoker address in README before merging.

**Comment by talbaneth** [comment]
@brianmcmichael @hexonaut please approve to merge.

File: README.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ Optimized Smart Contract to Poke (`poke`).
 
 For Now, Hard Coded Addresses and Sequences. Easy for TechOps to Run.
 
-MegaPoker curent Mainnet Address: [0x8e01D186594983943A614cC724bD1d6F8bFdd41B](https://etherscan.io/address/0x8e01D186594983943A614cC724bD1d6F8bFdd41B#code)
+MegaPoker curent Mainnet Address: [0x70098F537EE8D0E00882585b7B02C45cd6AB3186](https://etherscan.io/address/0x70098F537EE8D0E00882585b7B02C45cd6AB3186#code)
```
Comment (brianmcmichael) [inline README.md:8]: preexisting spelling error

```suggestion
MegaPoker current Mainnet Address: [0x70098F537EE8D0E00882585b7B02C45cd6AB3186](https://etherscan.io/address/0x70098F537EE8D0E00882585b7B02C45cd6AB3186#code)
```

**Pull request merged**